rapids-cmake/test/generate_resource_spec.cmake

```diff
@@ -60,7 +60,8 @@ function(rapids_test_generate_resource_spec DESTINATION filepath)
 
     set(compiler "${CMAKE_CXX_COMPILER}")
     set(lang CXX)
-    if(DEFINED CMAKE_CUDA_COMPILER)
+    get_property(rapids_languages GLOBAL PROPERTY ENABLED_LANGUAGES)
```

Comment:
Github isn't letting me add comments on lines that haven't been changed, but should probably amend the CUDA/C++ existence check on lines 49-52 as well with this.

Reply:
Fixed!